src/status_im/chat/models/message.cljs

```diff
@@ -147,6 +148,14 @@
                         assoc
                         :unviewed-messages-count (inc current-count))}))))
 
+(fx/defn check-for-incoming-tx
+  [cofx {{:keys [transaction-hash]} :command-parameters}]
+  (when (and transaction-hash
+             (not (string/blank? transaction-hash)))
+    {:dispatch-later
```

Comment:
could you add a comment?

Reply:
what comment do you want to have here? Explanation why it is dispatch later instead of direct call or something else?

Reply:
Explanation why it is dispatch later instead of direct call yes

Reply:
because of circular dependency basically, but ok :D